src/startupscriptgenerator/src/restorePackages.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+declare -r WORKSPACE_DIR=$( cd $( dirname "$0" ) && cd .. && pwd )
+tomlFileName="Gopkg.toml"
+
+echo "Restoring packages..."
+echo "Installing dep..."
+go get -u github.com/golang/dep/cmd/dep
+
+function runDepEnsure() {
+    pkgDir="$1"
+    if [ -f "$pkgDir/$tomlFileName" ]; then
+            echo "Running 'dep ensure' under '$pkgDir'..."
+            cd $pkgDir
+            dep ensure
+        else
+            echo "Cound not find '$tomlFileName' under '$pkgDir'. Not running 'dep ensure'"
+    fi
+}
+
+runDepEnsure $WORKSPACE_DIR/src/common
```

Comment:
restoring packages under each folder that we care (i say this because we can other folders under `src`...for example `github.com` `gitlang.org` etc.)

Reply:
this wouldn't be a problem if we stick them all under `vendor` :)